src/main/topLevelPages/topLevelSpaces/SpaceExplorerView.tsx

```
@@ -68,19 +70,23 @@ interface Space extends Identifiable {
 }
 
 const collectParentAvatars = <SpaceWithVisuals extends WithBanner & WithParent<WithBanner>>(
-  { about, parent }: SpaceWithVisuals,
+  { about, parent, id }: SpaceWithVisuals,
   initial: string[] = []
 ) => {
   if (!about?.profile) {
     return initial;
   }
 
   const { cardBanner, avatar = cardBanner } = about?.profile;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)